.github/ISSUE_TEMPLATE.md

```diff
@@ -1,4 +1,4 @@
-<!-- Please only file either bugs or any feature request that you would like to suggest for Foundation for sites on GitHub. If you've got a more general question about how to use Foundation, we can help you on the Foundation Forum: http://foundation.zurb.com/forum -->
```

Comment:
Ohk @Deckluhm .... moreover i by mistake did wrong branching (IamManchanda:master) so closing this and opening in another!